src/guide/1/getting-started/first-steps/snippets/subscribe.js

```diff
@@ -0,0 +1,39 @@
+// load the Kuzzle SDK module
+const {
+  Kuzzle,
+  WebSocket
+} = require('kuzzle-sdk');
+// instantiate a Kuzzle client
+const kuzzle = new Kuzzle(new WebSocket('kuzzle'));
+const message = {message: 'Hello, World!'};
+// define a filter
+const filter = {exists: {field: 'message'}};
+// will be called each time a document match the filter
+const callback = (notifications) => {
```

Comment:
(optional / nitpicking)

```suggestion
const callback = notifications => {
```